mycroft/util/lang/format_en.py

```diff
@@ -71,18 +72,34 @@
 }
 
 
-def nice_number_en(mixed):
+def nice_number_en(number, speech, denominators):
```

Comment:
The comment below isn't quite up to date with these changes, can you update that as well?

Reply:
It's just the description of the function, right ? The args and return value are up to date.
Also there's talk in the Issue #1411 that instead of moving the number formatting code to each language's code, we should just put a failsafe for every language in the main function in case it's not yet implemented.

Reply:
Yes, the args and return value lookes good I just meant the (1, 1, 3) part.

And, ah humm I see what you mean now. It might be more clear to have it structured like you have here and then we can have a simpler nice_number in format.py. unless the text formatting becomes trivial with the use of `format` you suggest.

Reply:
This use of <code>format</code> would actually mean the <code>nice_number</code> function would work for displays out of the box for every language and the language specific function would only be needed for speech to help the TTS.

Reply:
Sounds good!

Reply:
I just read on [python's locale doc](https://docs.python.org/2/library/locale.html#background-details-hints-tips-and-caveats) that it's not recommended to change the locale setting on the fly since it's expensive on resources and as a global value, it will affect the entire program, even temporarily.

Should it be set at startup when reading the conf file to allow us to use the format function for each language without worrying? Or should we just drop this for now and keep what's been done?